### DISCUSSION TITLE
[Feature] Face Detection - unmerge / split faces and re-evaluate faces

### DISCUSSION THREAD

**Discussion by dharapvj** [immich-app/immich#3356] [Feature Request]
### The feature

Currently, Immich is detecting faces of both my kids to be just one face!

there is no way for me to split them and mark two kids apart. Also - I would imagine - that when we allow such functionality - we will need ability to re-evaluate all faces marked as kid1 to be re-calculated so that kid2 can re-register all his/her faces correctly under its bucket.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2472

**Comment by ianfixes** [comment]
I think this should be reopened -- after reading #2472, I see many features but no mention of the word "split".

Also, given that #2472 is merged and I am running `v2.6.1`, I have a the same issue described here: two people (often times appearing in the same photo together) are clustered as the same person with no clear feature for splitting the recognition.

**Comment by ianfixes** [comment]
I've achieved some success with this tool
https://github.com/pabera/immich-face-fix

I'm going to try and use this as a base to develop more face-sorting functions:
- display all detected faces for one person, to look for incorrect tags
- show photos with a large number of unnamed faces (e.g. crowd shots) so I can bulk-ignore faces
- some general capabilities for saving and restoring face tags so I can try out different facial recognition settings without duplicating hours of manual tagging

**Comment by bo0tzz** [comment]
I think this is possible these days, you can edit the recognized faces on an asset and then remove or reassign them from there.
